components/Resources/ServicesSearchResults.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <div class="resources-search-results">
+    <div class="no-results" v-if="services.length === 0">
+      <el-table :show-header="false" empty-text="No Results">
+        <el-table-column />
+      </el-table>
+    </div>
+    <div
+      v-else
+      v-for="service in services"
+      :key="service.id"
+      class="resources-search-results__items"
+    >
+      <div class="resources-search-results__items--image">
+        <img v-show="service.thumbnail" :src="service.thumbnail" />
+      </div>
+      <div class="resources-search-results__items--content">
+        <h2><span v-html="highlightMatches(service.title, $route.query.search)"/></h2>
+        <p
+          class="resources-search-results__items--content-description"
+          v-html="highlightMatches(parseMarkdown(service.description), $route.query.search)"
+        />
+      </div>
+      <div  class="resources-search-results__items--button">
+        <sparc-tooltip placement="top-center" content="Instantiate this service in o²S²PARC. Login credentials are required.">
+          <div slot="item">
+            <a :href="service['view_url']" target="_blank">
+              <el-button>
+                <div class="open-osparc-btn-content">
+                  Open in o<sup>2</sup>S<sup>2</sup>PARC <svg-icon icon="icon-open" height="18" width="18" />
+                </div>
+              </el-button>
+            </a>
+          </div>
+        </sparc-tooltip>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import marked from '@/mixins/marked/index'
+import { highlightMatches } from '../../pages/data/utils'
+
+export default {
+  name: 'ServicesSearchResults',
+
+  mixins: [marked],
+
+  props: {
+    services: {
+      type: Array,
+      default: () => []
+    }
+  },
+
+  methods: {
+    highlightMatches
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/_variables.scss';
```

Comment:
Replace this with import with @import '@nih-sparc/sparc-design-system-components/src/assets/_variables.scss' and update variables accordingly (it looks like you should only have to update the colors using this for reference: https://nih-sparc.github.io/sparc-design-system-components/?path=/docs/components-colors--basic).

Also might as well do the same for ResourcesSearchResults.vue

i.e. $median -> $purple
      $light-grey -> $lineColor2
      $dark-sky -> $grey

Reply:
done (please check)